src/Model/Serializer/ValueObject/MediaObject/ImageDenormalizer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\Serializer\ValueObject\MediaObject;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUIDParser;
+use CultuurNet\UDB3\Model\ValueObject\MediaObject\CopyrightHolder;
+use CultuurNet\UDB3\Model\ValueObject\MediaObject\Image;
+use CultuurNet\UDB3\Model\ValueObject\MediaObject\MediaObjectIDParser;
+use CultuurNet\UDB3\Model\ValueObject\Text\Description;
+use CultuurNet\UDB3\Model\ValueObject\Translation\Language;
+use CultuurNet\UDB3\Model\ValueObject\Web\Url;
+use Symfony\Component\Serializer\Exception\UnsupportedException;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+final class ImageDenormalizer implements DenormalizerInterface
+{
+    /**
+     * @var UUIDParser
+     */
+    private $imageIdParser;
+
+    public function __construct(UUIDParser $imageIdParser = null)
+    {
+        if (!$imageIdParser) {
+            $imageIdParser = new MediaObjectIDParser();
+        }
+        $this->imageIdParser = $imageIdParser;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function denormalize($data, $class, $format = null, array $context = [])
+    {
+        if (!$this->supportsDenormalization($data, $class, $format)) {
```

Comment:
This is something we don't consistently verify. We should maybe add this to all denormalizers?

For example:
https://github.com/cultuurnet/udb3-silex/blob/master/src/Model/Serializer/ValueObject/Calendar/StatusDenormalizer.php#L21

Reply:
It could help in case someone injects an incorrect implementation of `DenormalizerInterface` somewhere. But in that case we'll probably get some other kind of error as well that would point to the issue. I'd say feel free to add it in new denormalizers and update existing ones if you change them anyway :)